theme/js/components/dataset/resource/resource.vue

```diff
@@ -186,26 +247,58 @@ export default defineComponent({
   setup(props) {
     const owner = useOwnerName(props.resource);
     const resourceImage = useResourceImage(props.resource);
+    const { getComponentsForHook } = useComponentsForHook();
     const filters = inject('$filters');
-    const showModal = inject('$showModal');
+    /** @type {import("vue").Ref<HTMLElement | undefined>} */
+    const content = ref();
     const expanded = ref(false);
-    const expand = () => expanded.value = !expanded.value;
+    const expand = () => {
+      expanded.value = !expanded.value;
+      if(content.value) {
+        toggleAccordion(content.value, expanded.value, 24);
+      }
+    }
     const availabilityChecked = computed(() => props.resource.extras && props.resource.extras['check:status']);
     const lastUpdate = computed(() => props.resource.published > props.resource.last_modified ? props.resource.published : props.resource.last_modified);
     const unavailable = computed(() => availabilityChecked.value && availabilityChecked.value >= 400);
-    const showSchemaButton = computed(() => props.resource.schema && props.resource.schema.name);
+    const { authorizeValidation, documentationUrl, loading, validationUrl} = useSchema(props.resource);
+    const explore = getComponentsForHook("explore");
+    const hasExplore = computed(() => explore.length > 0 && explorable_resources && explorable_resources.includes(props.resource.id));
```

Comment:
Not sure about this logic here, but we could start with this.

Alternatives:
* Having a `can_explore` property instead?
* Having the component having the responsibility directly and calling it in any case.
* ... ?

Reply:
Can we add the property from udata_tabular_preview ?

Regarding the 2nd alternative, we can make Explore component checks the explorable_resources config and dispatch an event that `resource.vue` can catch so it knows if the explore is shown or not.

Reply:
After discussion, we can keep with the actual logic.
In the future and based on user feedbacks on the explore/preview features, we will either keep with the distinction and add a dedicated `can_explore` property, or merge both concepts.